content/en/docs/cluster-administration/cluster-settings/log-collections/add-fluentd-as-receiver.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Add Fluentd as Receiver (aka Collector)"
+keywords: 'kubernetes, log, fluentd, pod, container, fluentbit, output'
+description: 'KubeSphere Installation Overview'
+
+linkTitle: "Add Fluentd as Receiver"
+weight: 2400
+---
+KubeSphere supports using Elasticsearch, Kafka and Fluentd as log receivers.
+This doc will demonstrate:
+
+- How to deploy Fluentd as deployment and create corresponding service and configmap.
+- How to add Fluentd as a log receiver to receive logs sent from Fluent Bit and then output to stdout.
+- How to verify if Fluentd receives logs successfully.
+
+## Prerequisite
+
+To configure log collection, you should use an account of ***platform-admin*** role.
+
```

Comment:
Add the following content under this part as well:

You need to enable Logging, Events or Auditing Logs by following [Enable Pluggable Components](https://kubesphere.io/docs/pluggable-components/). Logging is enabled in this example for demonstration.